frappe/utils/boilerplate.py

```diff
@@ -28,6 +28,9 @@ def make_boilerplate(dest, app_name):
 			hook_val = cstr(raw_input(key + ": "))
 			if hook_key=="app_name" and hook_val.lower().replace(" ", "_") != hook_val:
 				print "App Name must be all lowercase and without spaces"
+  				hook_val = ""
+			elif hook_key=="app_title" and not re.match("^(?![\W])[^\d_\s][\w -]+$", hook_val, re.UNICODE):
+				print "Invalid App Title"
```

Comment:
print "App Title should start with a letter and it can only consist of letters, numbers, spaces and underscores"